relay-general/src/store/transactions.rs

```diff
@@ -153,9 +152,7 @@ impl Processor for TransactionsProcessor {
             ));
         }
 
-        if span.op.value().is_none() {
-            return Err(ProcessingAction::InvalidTransaction("span is missing op"));
-        }
+        span.op.get_or_insert_with(|| "op".to_owned());
```

Comment:
This is where we set the default value for `op`.
Using `"op".to_owned()` to match the code style in the code base, as suggested by Jan.